hydroflow/src/lang/monotonic_map.rs

```diff
@@ -8,7 +6,6 @@ use super::clear::Clear;
 pub struct MonotonicMap<K, V>
 where
     K: PartialOrd,
-    V: Clear,
```

Comment:
@MingweiSamuel why did monotonicmap require V: Clear? Was it something left over from when it used to call clear? Seems nothing is depending on it implementing Clear anymore and for some reason FxHashSet does not implement Clear so I had to remove it.